docs/CONFIG-VARS.md

```diff
@@ -164,21 +164,22 @@ When setting V4_CFG_MANAGE_STORAGE to true, A new storage classes will be create
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
 | V4M_LOGGING_NAMESPACE | Namespace for the logging resources | string | logging | false | | cluster-logging |
-| V4M_KIBANA_FQDN | FQDN to use for kibana ingress | string | kibana.<V4M_BASE_DOMAIN> | false | | cluster-logging |
+| V4M_KIBANA_FQDN | FQDN to use for kibana ingress | string | dashboards.<V4M_BASE_DOMAIN> | false | | cluster-logging |
 | V4M_KIBANA_CERT | Path to tls certificate to use for kibana ingress | string |<V4M_CERT> | false | If both this and V4M_CERT are not set a self-signed cert will be used | cluster-logging |
 | V4M_KIBANA_KEY | Path to tls key to use for kibana ingress | string | <V4M_KEY> | false | If both this and V4M_KEY are not set a self-signed cert will be used | cluster-logging |
 | V4M_KIBANA_PASSWORD | Kibana admin password | string | randomly generated | false | If not provided, a random password will be generated and written to the log output | cluster-logging |
 | V4M_KIBANASERVER_PASSWORD | Kibana server password | string | randomly generated | false | If not provided, a random password will be generated and written to the log output | cluster-logging |
 | V4M_LOGCOLLECTOR_PASSWORD | Logcollector password | string | randomly generated | false | If not provided, a random password will be generated and written to the log output | cluster-logging |
 | V4M_METRICGETTER_PASSWORD | Metricgetter password | string | randomly generated | false | If not provided, a random password will be generated and written to the log output | cluster-logging |
 | | | | | | | |
-| V4M_ELASTICSEARCH_FQDN | FQDN to use for elasticsearch ingress  | string | elasticsearch.<V4M_BASE_DOMAIN> | false | | cluster-logging |
+| V4M_ELASTICSEARCH_FQDN | FQDN to use for elasticsearch ingress  | string | search.<V4M_BASE_DOMAIN> | false | | cluster-logging |
 | V4M_ELASTICSEARCH_CERT | Path to tls certificate to use for elasticsearch ingress | string |<V4M_CERT> | false | If both this and V4M_CERT are not set a self-signed cert will be used | cluster-logging |
 | V4M_ELASTICSEARCH_KEY | Path to tls key to use for elasticsearch ingress | string | <V4M_KEY> | false | If both this and V4M_KEY are not set a self-signed cert will be used | cluster-logging |
+| V4M_OSD_NODEPORT_ENABLE |  If you want to make OpenSearch Dashboards accessible via NodePort, set the environment variable V4M_OSD_NODEPORT_ENABLE to true. OpenSearch Dashboards will be accessible from port 31034 | bool | false | false | | cluster-logging
```

Comment:
Dashboards plural was intentional, the official name is "OpenSearch Dashboards" which Greg had advised us to use.
Here's a recent response from Greg re: this variable name:

[12:31 PM] Greg Smith
Hey David...I understand Thomas's concern.  However, since we've used "search" to mean "opensearch", I think it would be confusing to have the property called V4M_SEARCH_NODEPORT_ENABLE when it actually enables the nodeport for OpenSearch Dashboards (which is a separate application than OpenSearch).  The reason I suggested moving away from V4M_KB_KNOWN_NODEPORT_ENABLE name was to a) remove the reference to Kibana (i.e. the KB), and, b) shorten the already too long name.  We use the "osd" terminology within our solution to refer to OpenSearch Dashboards (e.g. the user-values-osd.yaml file you are creating).

[12:34 PM] Greg Smith
Ultimately, the configuration property names are a DAC thing so I suppose you guys are free to name them as you see fit.  I suppose the simplest thing is to prepend V4M_ to any of "our" environment variable names.  But, in this case, I know we will probably drop the KB_KNOWN_NODEPORT_ENABLE env var at some point and replace it with something like OSD_KNOWN_NODEPORT_ENABLE or OSD_NODEPORT_ENABLE.

[12:35 PM] Greg Smith
If you do decide to change the name, the UDANext team will need to know so they can update their code (which, as I understand it, is expecting the V4M_OSD_NODEPORT_ENABLE name).